packages/client/src/generation/TSClient/PrismaClient.ts

```diff
@@ -312,22 +312,23 @@ function interactiveTransactionDefinition(this: PrismaClientClass) {
   }
 
   const returnType = ts.promise(ts.namedType('R'))
+
+  const thisType = ifExtensions<ts.TypeBuilder>(
```

Comment:
`this` refers to `PrismaClient` type without client component applied (normally, we add later via `GetClient` type), so methods added through that type of extensions were not available to itx client.